### PR TITLE
Random odd in java

### LINKED ISSUE #1120: Write a Java programme to generate a random odd number in range n1-n2
### Description

Write a programme to generate a random odd number in range n1-n2

```
Start range : 0
End   range : 100

Output : 33
```

### LINKED ISSUE #1137: Write a C programme to find index of an alphabet
### Description

Write a programme to find index of an alphabet

> **Note** index starts from 1

```
Input  : C
Output : 3
```

### PR DESCRIPTION
**Pull request by SpirosArk**
### Why:

Closes #1120 
Closes #1137 

### Check off the following:

- [ x] I have reviewed my changes in staging (look for "Automatically generated comment" and click **Modified** to view your latest changes).
- [x ] For content changes, I have completed the [self-review checklist](https://github.com/codinasion/codinasion/blob/master/contributing/self-review.md).

### CODE REVIEW

File: programme/index-of-alphabet/index-of-alphabet.c [hunk 1]
```diff
@@ -0,0 +1,24 @@
+//Program that accepts a char as an input and returns it's position in alphabet
```
Comment (KikiManjaro) [inline programme/index-of-alphabet/index-of-alphabet.c:1]:
```suggestion
//Programme that accepts a char as an input and returns it's position in alphabet
```

File: programme/random-odd-number/random-odd-number.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
+//Program to generate random odd number in range using Java
```
Comment (KikiManjaro) [inline programme/random-odd-number/random-odd-number.java:1]:
```suggestion
//Programme to generate random odd number in range using Java
```

File: programme/volume-of-cylinder/volume-of-cylinder.jl [hunk 3]
```diff
@@ -0,0 +1,8 @@
+#Program to calculate volumeOfCylinder
```
Comment (KikiManjaro) [inline programme/volume-of-cylinder/volume-of-cylinder.jl:1]:
```suggestion
#Programme to calculate volumeOfCylinder
```

**Review by KikiManjaro — CHANGES_REQUESTED** [review]
Minor fixes related to #1295

File: programme/random-odd-number/random-odd-number.java [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+        System.out.println("Enter end range:");
+        int endRange = s_name.nextInt();
+        
+        System.out.println("Random value in int from "+startRange+" to "+endRange+ ":\n");
+        int random_int = (int)Math.floor(Math.random()*(endRange-startRange+1)+startRange);
+        while (random_int % 2 == 0){
+            
+            random_int = (int)Math.floor(Math.random()*(endRange-startRange+1)+startRange);
+            
+        }
```
Comment (KikiManjaro) [inline programme/random-odd-number/random-odd-number.java:21]:
```suggestion
        while (random_int % 2 == 0){
            random_int = (int)Math.floor(Math.random()*(endRange-startRange+1)+startRange);
        }
```

File: programme/index-of-alphabet/index-of-alphabet.c [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+    if (input >= 97){
+        
+        position = input - 'a' + 1;
+        
+    }
+    else{
+    
+        position = input - 'A' + 1;
+    
+    }
```
Comment (KikiManjaro) [inline programme/index-of-alphabet/index-of-alphabet.c:20]:
```suggestion
    if (input >= 97){
        position = input - 'a' + 1;
    }
    else{
        position = input - 'A' + 1;
    }
```

**Comment by KikiManjaro** [comment]
You've made a mistake and modified index-of-alphabet and volume in this MR, this one should only refer to Random odd

**Review by harshraj8843 — APPROVED** [review]
👍🏻

**Comment by harshraj8843** [comment]
Hey @SpirosArk , please consider different solutions in different PR(s) 👍🏻

**Pull request merged**